nautobot/docs/release-notes/version-1.6.md

```diff
@@ -76,6 +76,7 @@ As Python 3.7 has reached end-of-life, Nautobot 1.6 and later do not support ins
 
 ### Security
 
+- [#5911](https://github.com/nautobot/nautobot/issues/5911) - Updated `zipp` to `3.19.1` to address `CVE-2024-5569`. This is not a direct dependency so it will not auto-update when upgrading. Please be sure to upgrade your local environment.
```

Comment:
No update to `poetry.lock` for this one?

Reply:
`zipp` is on 3.19.2 already in poetry.lock